src/assets/static/robots.txt.ejs

```diff
@@ -0,0 +1,152 @@
+# The FULL URL to the DSpace sitemaps
+# The dspaceUrl will be auto-filled with the value in environment.rest.baseUrl
+# XML sitemap is listed first as it is preferred by most search engines
+Sitemap: <%= dspaceUrl %>/sitemaps/sitemap0.xml
+Sitemap: <%= dspaceUrl %>/sitemaps/sitemap0.html
```

Comment:
I *think* the Sitemap will eventually need to "appear" as if it is also coming from the UI (rather than linking back to the REST API).  From my understanding Sitemaps are not supposed to cross domains (and it is possible that the REST API and UI will be on separate domains).

But, that said, we can always clean this up later on in a followup PR as needed.  It's not directly related to the idea of making a robots.txt file possible.

Reply:
If we proxy the sitemap, there's no longer any need to have robots.txt be an ejs template, which in turn will fix the content-type issue you pointed out.

So I opted to fix it in this PR